ui/page/receive/receive_page.go

```diff
@@ -42,16 +41,18 @@ type Page struct {
 	*app.GenericPageModal
 	modalLayout *cryptomaterial.Modal
 
-	pageContainer         layout.List
-	scrollContainer       *widget.List
-	isNewAddr             bool
-	currentAddress        string
-	qrImage               *image.Image
-	newAddr, copy         *cryptomaterial.Clickable
-	info                  cryptomaterial.IconButton
-	card                  cryptomaterial.Card
-	sourceAccountselector *components.WalletAndAccountSelector
-	sourceWalletSelector  *components.WalletAndAccountSelector
+	pageContainer   layout.List
+	scrollContainer *widget.List
+	isNewAddr       bool
+	currentAddress  string
+	qrImage         *image.Image
+	newAddr, copy   *cryptomaterial.Clickable
+	info            cryptomaterial.IconButton
+	card            cryptomaterial.Card
+
+	walletDropdown     *components.WalletDropdown
```

Comment:
why did you remove `components.WalletAndAccountSelector`?

Reply:
It has already been replaced by walletDropdown and accountDropdown